configure.in

```diff
@@ -221,6 +221,7 @@ AC_CHECK_LIB(grpc, main)
 AC_CHECK_LIB(grpc++, main)
 AC_CHECK_LIB(grpc++_reflection, main)
 AC_CHECK_LIB(tkrzw_rpc, main, AC_MSG_WARN([old version of Tkrzw-RPC was detected])) 
+AC_CHECK_LIB(absl_synchronization, main)
```

Comment:
As this is just an aesthetic viewpoint, could you put this line at line 220 (between the rules for grpc and gpr)?
I'd like to sort the rules in order from lower to higher in terms of dependency.

Reply:
Sure,
A.